src/events/qaMention.js

```diff
@@ -1,11 +1,12 @@
 const { Events } = require('discord.js');
 const { QA_MENTION } = require('../config');
+const { translateLanguage } = require('../languages');
 
 module.exports = {
   name: Events.MessageCreate,
   async execute(message) {
     try {
-      if (!message || !message.author || message.author.bot) {
+      if (!message?.author?.bot || !message.guild) {
```

Comment:
```suggestion
if (!message?.author?.bot || !message?.guild) {
```

Reply:
Just in case seems like this change wasn't submitted